app/gui/src/controller/project.rs

```diff
@@ -12,44 +12,13 @@ use engine_protocol::language_server::Path;
 use enso_frp::web::platform;
 use enso_frp::web::platform::Platform;
 use parser::Parser;
-use semver::Version;
-use semver::VersionReq;
-use serde::Deserialize;
-use serde::Serialize;
 
 
 
-/// Application config.
```

Comment:
I'd rather unify config handling in favor of the here removed code - that is define structure in Rust and fill it from config, rather than generating structure from js. Can be tracked as separate issue (if agreed upon)